compiler/modules/comm.diagnostic/diagnostic.bal

```diff
@@ -25,6 +25,7 @@ public type File readonly & object {
     public function filename() returns string;
     public function directory() returns string?;
     public function lineColumn(Position pos) returns LineColumn;
+    public function getRange(Range|Position range) returns string[];
```

Comment:
This interface should provide a function that returns the string containing a particular Position. Its name should not start with `get` because none of the other methods do.  We have two choices:

1. rename lineColumn and related functions to use `number` in their names, and then call this one `line`
2. call this one something like `lineContent`.

Let's do 2 in this PR, and maybe consider doing 1 later.

Reply:
fixed